### ISSUE TITLE
Respect dot as a word separator

### ISSUE THREAD

**Issue by ildar2** [wuseal/JsonToKotlinClass#249]
```
@SerializedName("mobile.agree_with_conditions")
val mobileagreeWithConditions: String = "",
```
in my localization json most of the lines have "mobile." prefix. It gets skipped when field names are generated.
Could you make dot "." a valid word separator, so `mobileagreeWithConditions` would become `mobileAgreeWithConditions`?
P.S. please check out my comment at #84

**Comment by wuseal** [comment]
@ildar2 Of course, Let me check later

**Comment by wuseal** [comment]
@ildar2 As u sad, you indeed need it, So let's release it this week:grinning:

**Comment by wuseal** [comment]
https://github.com/wuseal/JsonToKotlinClass/releases/download/3.5.1-EAP/JsonToKotlinClass-3.5.1.zip
New EAP version has been release have a try if it works for you @ildar2

**Comment by ildar2** [comment]
Nice! It's doing exactly what I need! Thank you!
Since I created this issue I learned regex replace and managed to work around it with replace mobile([a-z]) to mobile\u$1\E :) now I dont have to, thanks to you

The only limitation I met is java constructor can have 256 parameters max, so I can't have all 500 fields as constructor parameters in data class. That is easy to fix manually tho. Again, thank you, I will continue using your plugin!
